cmd/endpoint.go

```diff
@@ -142,6 +147,16 @@ func NewEndpoint(arg string) (ep Endpoint, e error) {
 	}, nil
 }
 
+// checkIPv4 - helper for validating if the provided arg is an ip address.
+func checkIPv4(ipAddress string) bool {
```

Comment:
1. You could name this function as `isHostIPv4()` or `isHostIP4()`. Function naming `check` usually return error.
2. As given arg is split, you could use `serverAddr` than `ipAddress`.  This means it doesn't receive ip address.
3. you could consider moving this function to net.go